common/persistence/nosql/nosql_execution_store_test.go

```diff
@@ -34,108 +34,179 @@ import (
 	"github.com/uber/cadence/service/history/constants"
 )
 
-func TestCreateWorkflowExecution(t *testing.T) {
-	ctrl := gomock.NewController(t)
-	defer ctrl.Finish()
-
-	mockDB := nosqlplugin.NewMockDB(ctrl)
-	logger := log.NewNoop()
-
-	store := &nosqlExecutionStore{
-		shardID:    1,
-		nosqlStore: nosqlStore{logger: logger, db: mockDB},
-	}
-
+func TestNosqlExecutionStore(t *testing.T) {
 	ctx := context.Background()
-	request := &persistence.InternalCreateWorkflowExecutionRequest{
-		RangeID:                  123,
-		Mode:                     persistence.CreateWorkflowModeBrandNew,
-		PreviousRunID:            "previous-run-id",
-		PreviousLastWriteVersion: 456,
-		NewWorkflowSnapshot:      getNewWorkflowSnapshot(),
-	}
+	shardID := 1
 	testCases := []struct {
 		name          string
-		setupMock     func()
-		request       *persistence.InternalCreateWorkflowExecutionRequest
+		setupMock     func(*gomock.Controller) *nosqlExecutionStore
+		testFunc      func(*nosqlExecutionStore) error
 		expectedError error
 	}{
 		{
-			name: "success",
-			setupMock: func() {
+			name: "CreateWorkflowExecution success",
+			setupMock: func(ctrl *gomock.Controller) *nosqlExecutionStore {
+				mockDB := nosqlplugin.NewMockDB(ctrl)
 				mockDB.EXPECT().
 					InsertWorkflowExecutionWithTasks(gomock.Any(), gomock.Any(), gomock.Any(), gomock.Any(), gomock.Any(), gomock.Any(), gomock.Any(), gomock.Any()).
```

Comment:
I don't think there's a huge amount of value in this kind of test unless we do some assertion on what's being passed to the DB (I don't know this function particularly well). I get that here though you're just adding to a prexxisting setup

Reply:
yes! Adding to a preexisting setup just to increase coverage.